cart&wishlist/cart.css

```diff
@@ -0,0 +1,111 @@
+
+
+/* card component  */
+
+.footer{
+    justify-content: space-between;
+    padding: 1rem 1.5rem;
+    align-items: center;
+}
+
+.footer button{
+    color: #0346fcbd;
+    font-size: 1.1rem;
+    border:none;
+    background-color: transparent;
+    text-transform: uppercase;
+    font-weight: 700;
+    padding: 0.2rem;
+    cursor: pointer;
+}
+
+
+.footer .button-icon {
+    width: 40%;
+    justify-content: space-around;
+    align-items: center;
+    cursor: pointer;
+  }
+
+  .footer .button-icon button {
+    color: rgba(0, 0, 0, 0.6);
+    height: 30px;
+    width: 30px;
+    border-radius: 50%;
+    font-weight: normal;
+    
+  }
+
+
+
+.card-hz{
+    max-width: 350px;
+    border: 1px solid #f7f7f8;
+    border-radius: 0.5rem;
+    box-shadow:rgba(0,0,0,0.3)0px 3px 8px ;
+  }
+
+  .card-hz .card-header{
+    display: flex;
+    /* flex-basis: 70%; */
+    /* overflow: hidden; */
+
+  }
+  
+  .card-hz .card-header-img {
+    object-fit: cover;
+    flex-basis: 50%;
+    width: 80px;
+    height: 100px;
```

Comment:
here you want to use **rem** if the value is less than 1rem then use **px** the best practice is rem as Vikram told us.